LSE-61.tex

```diff
@@ -2635,13 +2794,13 @@ \subsubsection{Data Access Services}
 \textbf{ID:} DMS-REQ-0364
 
 \textbf{Specification:}
-The data access services shall be designed to permit, and their software implementation shall support, the service of at least \textbf{nDRTot} Data Releases accumulated over the (find the actual survey-length parameter) \textbf{surveyYears}-year planned survey.
+The data access services shall be designed to permit, and their software implementation shall support, the service of at least \textbf{nDRTot} Data Releases accumulated over the (find the actual survey-length parameter) \textbf{surveyYears}-year planned survey.</body>
```

Comment:
The `</body>` seems like a document generation glitch?

Reply:
I think HTML output changed in MagicDraw 19. I'm fixing my post-processor.